### ISSUE TITLE
Edit table data does not work when primary key column has uppercase letters

### ISSUE THREAD

**Issue by jonaskello** [OmniDB/OmniDB#278]
First of all, this tool looks really promising, thanks for providing it!

I'm using Postgres server 9.5.2, and OmiDB 2.2.0 on mac desktop. I connect to my database and it has a table that is defined like this:

```
-- Table: dbo."Account"

-- DROP TABLE dbo."Account";

CREATE TABLE dbo."Account"
(
  "Id" uuid NOT NULL DEFAULT '00000000-0000-0000-0000-000000000000'::uuid,
  "AccountProvider" smallint NOT NULL DEFAULT 0,
  "ProviderUserName" character varying(250) NOT NULL DEFAULT ''::character varying,
  "PasswordHash" character varying(250),
  "PasswordSalt" character varying(250),
  "User_Id" uuid,
  CONSTRAINT "PK_dbo.Account" PRIMARY KEY ("Id"),
  CONSTRAINT "FK_dbo.Account_dbo.User_User_Id" FOREIGN KEY ("User_Id")
      REFERENCES dbo."User" ("Id") MATCH SIMPLE
      ON UPDATE NO ACTION ON DELETE NO ACTION
)
WITH (
  OIDS=FALSE
);
ALTER TABLE dbo."Account"
  OWNER TO postgres;

-- Index: dbo."Account_IX_User_Id"

-- DROP INDEX dbo."Account_IX_User_Id";

CREATE INDEX "Account_IX_User_Id"
  ON dbo."Account"
  USING btree
  ("User_Id");
```

When I right-click it and select Data Actions > Edit data I get this:

![image](https://user-images.githubusercontent.com/664076/32143721-38397682-bcae-11e7-948e-0fd993c6e788.png)

**Comment by wind39** [comment]
Hi @jonaskello ,

Thanks for reporting this. This bug is already handled and the fix will be included in 2.3, to be released very soon. Stay tuned!

![image](https://user-images.githubusercontent.com/7719686/32178194-47fca61e-bd73-11e7-849c-9287916bd4fd.png)

**Comment by rafaelthca** [comment]
Fixed in 2.3.0. Closing this.
